packages/shared/styles/components/organisms/SfHeader.scss

```diff
@@ -1,12 +1,27 @@
 @import "../../helpers";
 .sf-header-navigation-item {
-  flex: 1;
+  position: var(--header-navigation-item-position, relative);
   display: flex;
+  flex: var(--header-navigation-item-flex, 0);
+  @include for-desktop {
+    --header-navigation-item-flex: 1;
+    --header-navigation-item-position: static
+  }
+  &__content {
+    position: var(--header-navigation-item-content-position, fixed);
+    top: var(--header-navigation-item-content-position, 0);
+    width: var(--header-navigation-item-content-width, 100%);
+    z-index: var(--header-navigation-item-content-position, 2);
+    --header-navigation-item-padding: 0;
+  }
+  &__link {
+    --link-text-decoration: var(--header-navigation-item, none)
```

Comment:
I wonder if these should be --header-navigation-item-text-decoration?

Reply:
totally agree :)